spec/venue_spec.rb

```diff
@@ -44,12 +39,42 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  # Iteration 3
+
+  describe '#over_capacity?' do
+    it 'checks if patron size is over capcity (bool)' do
+      venue = Venue.new('Watermelon', 3)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('Josh')
+
+      expect(venue.over_capacity?).to be true
+    end
+
+    # Iteration 4
+
+  describe '#kick_out' do
+    it 'removes patrons until under or at capacity' do
+      venue = Venue.new('Soeu', 3)
+      venue.add_patron('Joe')
```

Comment:
Please follow the interaction pattern and what is given to you. We are also building the habits of paying attention to details.